scripts/upload-apk.sh

```diff
@@ -13,28 +13,36 @@ git clone --quiet --branch=apk https://fossasia:$GITHUB_API_KEY@github.com/fossa
 
 cd apk
 
-\cp -r ../app/build/outputs/apk/*/**.apk .
-\cp -r ../app/build/outputs/apk/debug/output.json debug-output.json
-\cp -r ../app/build/outputs/apk/release/output.json release-output.json
+\cp -r ../app/build/outputs/apk/*/*/**.apk .
+\cp -r ../app/build/outputs/apk/fdroid/debug/output.json fdroid-debug-output.json
```

Comment:
Also the jsons can be removed @CloudyPadmal

Reply:
Yes good catch. Only the APKs should be in the branch. @neel1998 can you update the PR with the change?